data/core.yaml

```diff
@@ -1000,7 +1000,7 @@ en:
       location: 'This feature was moved by both you and {user}.'
       nodelist: 'Nodes were changed by both you and {user}.'
       memberlist: 'Relation members were changed by both you and {user}.'
-      tags: 'You changed the <b>{tag}</b> tag to "{local}" and {user} changed it to "{remote}".'
+      tags: 'You changed the {tag} tag to "{local}" and {user} changed it to "{remote}".'
```

Comment:
It looks like the code in merge_remote_changes.js hasn’t been changed to wrap `{tag}` in `<b>`; is that intentional? A fine alternative would be to use plain quotation marks here instead, just like for the `{local}` and `{remote}` placeholders.

Reply:
I changed this because this was the only string that included direct html formatting, and thought that the extra bolding was not really important in this case. I added some quotes to make it consistent with the rest of the message